js_modules/dagster-ui/packages/ui-core/src/selection/SelectionAutoComplete.ts

```diff
@@ -33,7 +32,19 @@ import {
 } from './generated/SelectionAutoCompleteParser';
 import {SelectionAutoCompleteVisitor as _SelectionAutoCompleteVisitor} from './generated/SelectionAutoCompleteVisitor';
 
-type Suggestion = {text: string; displayText?: string};
+export type Suggestion =
+  | {
+      text: string;
+      displayText?: string;
+      type: 'function' | 'logical_operator' | 'parenthesis';
+    }
+  | {
+      text: string;
+      displayText?: string;
+      type: 'attribute';
+      attributeName?: string;
+    };
```

Comment:
This is in preparation for adding icons and other useful information to the results list